lib/options.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+var fs = require('fs');
+
+var defaultOptionsPath = 'test/mocha.opts';
+
+var options = Object.create(null, {
```

Comment:
just so it doesn't get lost, we don't need to set properties and use `Object.create()`; let's just make a plain old object.

it's a private API, and if someone is going to monkey with it, then that's their problem.  we don't need to lock down the values via `writable: false`.  IIRC there isn't a precedent for this in the codebase.